templates/telemetry/update/update.4.parquetmr.txt

```diff
@@ -56,8 +57,10 @@ message update {
     optional binary targetChannel (UTF8);
     optional binary targetVersion (UTF8);
     optional binary targetBuildId (UTF8);
+    optional binary targetDisplayVersion (UTF8);
```

Comment:
This field is not available in the incoming `update`  pings, as far as I can tell. A client-side code change would be required to add this field.